docs/install/index.md

```diff
@@ -38,16 +38,25 @@ Scaffold your InvenioRDM instance. Replace ``<version>`` with the version you wa
 - LTS release (for production systems): ``v9.1``
 - STS release (for feature previews): ``v11.0``
 
-```
+```console
 invenio-cli init rdm -c <version>
 # e.g:
 invenio-cli init rdm -c v9.1
 ```
 
 You will be asked several questions. If in doubt, choose the default.
 
+
 #### [4. Build, setup and run](build-setup-run.md)
 
+Now the scaffoling is complete, it is time to check the development requirements
+```console
+cd my-site/
+invenio-cli check-requirements --development
+```
+This will check the python, nodejs and npm versions that depend on the version of InvenioRDM that has been selected in the previous step
```

Comment:
```suggestion
```

Meh :smile: we can leave that sentence out altogether since we are checking for a longer list of system requirements than that (git, imagemagick, docker, docker-compose...) and keeping it in sync is unnecessary work.

Reply:
This sounds good. Removing the sentence completely